src/crs/projected.jl

```diff
@@ -82,7 +82,7 @@ function indomain(C::Type{<:Projected}, (; lat, lon)::LatLon)
 end
 
 Base.isapprox(coords₁::Projected{Datum}, coords₂::Projected{Datum}; kwargs...) where {Datum} =
-  isapprox(convert(Cartesian, coords₁), convert(Cartesian, coords₂); kwargs...)
+  isapprox(convert(Cartesian3D, coords₁), convert(Cartesian3D, coords₂); kwargs...)
 
 Base.isapprox(coords₁::Projected{Datum₁}, coords₂::Projected{Datum₂}; kwargs...) where {Datum₁,Datum₂} =
```

Comment:
can this method also use the `Cartesian3D` alias?

Reply:
Yes, it can.